dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -120,6 +120,16 @@ public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceC
         application.run("db", "migrate", configPath, "--include", "1.5.0");
     }
 
+    public static void runMigration(List<String> migrationList, Application<DockstoreWebserviceConfiguration> application, String configPath) {
+        migrationList.forEach(migration -> {
```

Comment:
Good, note that this can simplify much of cleanStatePrivate1, cleanStatePrivate2, etc.